voluncheer/login/tests.py

```diff
@@ -0,0 +1,14 @@
+from django.test import TestCase
```

Comment:
`voluncheer/login/...` no longer exists, please rebase the commit to reflect the updated code base.